doc/architecture/decisions/0021-BLAST-sequence-formats.md

```diff
@@ -0,0 +1,94 @@
+# 21. BLAST sequence formats
+
+Date: 2022-10-17
+
+## Status
+
+Draft
+
+## Context
+This document summarises different sequence formats supported by Ensembl BLAST
+
+## Supported formats
+We support the most common sequence formats for any BLAST program - Fasta format or sequence as plain text
+
+## Decision
+Like other blast search providers we also support 2 ways of input - by pasting sequence in the input box or by uploading a file.
+
+- Input sequence to be accepted in either Fasta format characters or as plain text format (bare sequences). (Sequence IDs may be supported in the future)
```

Comment:
Unify the way we spell FASTA across these documents. Most sources spell it FASTA. I can't remember if, historically, it was FastA. Whichever format you pick (FASTA, FastA, Fasta, fasta), we should use it consistently throughout.